prow/cmd/checkconfig/main_test.go

```diff
@@ -1252,10 +1252,25 @@ func TestValidateInRepoConfig(t *testing.T) {
 			prowYAMLFileName = tempFile.Name()
 		}
 
-		cfg := &config.Config{
-			ProwConfig: config.ProwConfig{PodNamespace: "my-ns"},
+		// Need an empty file to load the config from so we go through its defaulting
+		tempConfig, err := ioutil.TempFile("/tmp", "prow-test")
```

Comment:
`"/tmp"` also won't work on Windows. Based on https://golang.org/pkg/io/ioutil/#TempFile it sounds like you should use `""`.